### DISCUSSION TITLE
使用rmlui开发PC应用

### DISCUSSION THREAD

**Discussion by oahzuw** [ejoy/ant#76] [Ideas]
rmlui使用html画界面简单轻量级，如果能打包PC各平台二进制，开发应用不是吊打其他GUI

**Comment by cloudwu** [comment]
RmlUI 制作 PC 上的应用还远远比不上集成一个完整功能的浏览器。游戏界面和应用界面还是有很大的不同的。

**Comment by actboy168** [comment]
用来制作轻量级应用理论上可行，但是由于没有考虑过这方面的使用，所以离开箱即用还有很大的距离。需要你有较强的动手能力。

虽然ant是一个3d引擎，但我个人认为用它来制作跨平台的命令行工具也是相当的好用，bee提供了常用os api，ltask可以很简单的实现多任务并发等。
